mon-pix/config/optional-features.json

```diff
@@ -1,3 +1,6 @@
 {
-  "jquery-integration": true
+  "application-template-wrapper": false,
+  "default-async-observers": true,
+  "jquery-integration": true,
+  "template-only-glimmer-components": true
```

Comment:
En description de la PR, je croyais qu'on ne forçait pas le passage aux Glimmer Components tout de suite 🤔

Reply:
Discussion vérifiée dans la [doc](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/) (ping @laura-bergoens) 

> In our new setup, the parent component has an action which sets the new value. We pass this action to the child component, and the child component directly assigns it to the click of the button, using the {{on}} modifier. It also passes the value we want to call the @onClick using the fn helper. We don't need any additional logic in the child class itself - in fact, this could become a template-only component at this point.

Du coup, il faut comprendre le nom de variable plutôt comme `(template-only)-glimmer-components`

On le laisse à `true` pour suivre les recommandations Octane et commencer à construire selon celle ci

Reply:
ok